README.md

```diff
@@ -32,13 +33,25 @@ with a vcs file to produce an svg waveform diagram. Optional arguments include:
 Display all wires starting at time 4 and ending at wire `clk`'s tenth tick:
 
 ```bash
-sootty example/example3.vcd -s "time 4" -e "acc clk == const 10" -w "clk,rst_n,pc,inst" -d
+sootty -f "example/example3.vcd" -s "time 4" -e "acc clk == const 10" -w "clk,rst_n,pc,inst" -d
 ```
 
 Display wires `Data` and `D1` for 8 units of time starting when `Data` is equal to 20:
 
 ```bash
-sootty example/example1.vcd -l 8 -s "Data == const 20" -w "D1,Data" -d
+sootty -f "example/example1.vcd" -l 8 -s "Data == const 20" -w "D1,Data" -d
+```
+
+Saving a query for future use:
```

Comment:
Ideally the user would be able to save and reload queries without worrying about the implementation details. It could be nice to allow a blank flag, i.e.: `sootty -f "file.vcd" -S` which would store the query in a random key, and `sootty -f "file.vcd" -R` which would reload the query from the previous key.

Reply:
So a temp file that they don't intend to use later?